resources/ts/Components/Integrations/Detail/UrlList.tsx

```diff
@@ -159,40 +230,107 @@ export const UrlList = ({
               )}
             </div>
           ) : (
-            <FormElement
-              key={option.env}
-              label={`${t(`details.integration_settings.${option.env}`)}`}
-              component={
-                <div className="flex gap-2">
-                  <Input
-                    type="text"
-                    name="url"
-                    className="md:min-w-[40rem]"
-                    disabled={disabled}
-                    onChange={(e) =>
-                      onChangeNewUrl({
-                        environment: option.env,
-                        url: e.target.value,
-                        type,
-                      })
-                    }
-                  />
-                </div>
-              }
-            />
+            <div key={option.env} className="flex flex-col gap-2">
+              <FormElement
+                label={`${t(`details.integration_settings.${option.env}`)}`}
```

Comment:
You can simplify this a bit:
```suggestion
                label={t(`details.integration_settings.${option.env}`)}
```

Reply:
I no longer find this code